tests/test_zarr_encryption_ipfs.py

```
@@ -121,29 +161,38 @@ def test_upload_then_read(random_zarr_dataset: tuple[str, xr.Dataset]):
+        # Check if the values of 'temp' and 'precip' are equal in all datasets
+        assert np.array_equal(loaded_ds1["temp"].values, expected_ds["temp"].values), (
+            "Temp values in loaded_ds1 and expected_ds are not identical!"
+        )
+        assert np.array_equal(loaded_ds1["precip"].values, expected_ds["precip"].values), (
+            "Precip values in loaded_ds1 and expected_ds are not identical!"
+        )
+
+    # Open a new instance outside the context
+    loaded_ds2 = xr.open_zarr(store=hamt1_read_z3, consolidated=False)
+    with pytest.raises(ValueError):
+        test_read = loaded_ds2["temp"].values
```

Ruff (0.8.2)

Local variable `test_read` is assigned to but never used

Remove assignment to unused variable `test_read`

(F841)